tests/test_example_dags.py

```diff
@@ -79,7 +79,7 @@ def get_dag_bag() -> DagBag:
             file.writelines(["example_cosmos_sources.py\n"])
         if DBT_VERSION < Version("1.6.0"):
             file.writelines(["example_model_version.py\n"])
-            file.writelines(["example_clone.py\n"])
+            file.writelines(["example_operators.py\n"])
```

Comment:
The test file name which we want to ignore is `example_operators` not sure why it CI did not catch it.

Reply:
DAG file: https://github.com/astronomer/astronomer-cosmos/blob/main/dev/dags/example_operators.py

Reply:
It's probably because it adds to the `.airflowignore`, and Airflow doesn't try to see if what is in the ignore file exists but just does the diff between the actual files and the ones on the ignore list.